### PR TITLE
Introduce Docker and Docker Compose resources for WSO2 Stream Processor v4..4.x

### PR DESCRIPTION
**Pull request by chirangaalwis**
https://github.com/wso2/docker-sp/issues/81, fixes https://github.com/wso2/docker-sp/issues/82, fixes https://github.com/wso2/docker-sp/issues/83, fixes https://github.com/wso2/docker-sp/issues/84, fixes https://github.com/wso2/docker-sp/issues/87, fixes https://github.com/wso2/docker-sp/issues/88 and fixes https://github.com/wso2/docker-sp/issues/90.

### CODE REVIEW

File: dockerfiles/centos/worker/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set to latest Ubuntu LTS
-FROM wso2sp-base:4.3.0-centos
+FROM wso2sp-base:4.4.0-centos
```
Comment (DilanUA) [inline dockerfiles/centos/worker/Dockerfile]: change the comment to suit the OS

File: dockerfiles/centos/manager/Dockerfile [hunk 2]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set to latest Ubuntu LTS
-FROM wso2sp-base:4.3.0-centos
+FROM wso2sp-base:4.4.0-centos
```
Comment (DilanUA) [inline dockerfiles/centos/manager/Dockerfile]: change the comment to suit the OS

File: dockerfiles/centos/editor/Dockerfile [hunk 3]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set to latest Ubuntu LTS
-FROM wso2sp-base:4.3.0-centos
+FROM wso2sp-base:4.4.0-centos
```
Comment (DilanUA) [inline dockerfiles/centos/editor/Dockerfile]: change the comment to suit the OS

File: dockerfiles/centos/dashboard/Dockerfile [hunk 4]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set to latest Ubuntu LTS
-FROM wso2sp-base:4.3.0-centos
+FROM wso2sp-base:4.4.0-centos
```
Comment (DilanUA) [inline dockerfiles/centos/dashboard/Dockerfile]: change the comment to suit the OS

File: dockerfiles/alpine/worker/Dockerfile [hunk 5]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set to latest Ubuntu LTS
-FROM wso2sp-base:4.3.0-alpine
+FROM wso2sp-base:4.4.0-alpine
```
Comment (DilanUA) [inline dockerfiles/alpine/worker/Dockerfile]: change the comment to suit the OS

File: dockerfiles/alpine/manager/Dockerfile [hunk 6]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set to latest Ubuntu LTS
-FROM wso2sp-base:4.3.0-alpine
+FROM wso2sp-base:4.4.0-alpine
```
Comment (DilanUA) [inline dockerfiles/alpine/manager/Dockerfile]: change the comment to suit the OS

File: dockerfiles/alpine/editor/Dockerfile [hunk 7]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set to latest Ubuntu LTS
-FROM wso2sp-base:4.3.0-alpine
+FROM wso2sp-base:4.4.0-alpine
```
Comment (DilanUA) [inline dockerfiles/alpine/editor/Dockerfile]: change the comment to suit the OS

File: dockerfiles/alpine/dashboard/Dockerfile [hunk 8]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set to latest Ubuntu LTS
-FROM wso2sp-base:4.3.0-alpine
+FROM wso2sp-base:4.4.0-alpine
```
Comment (DilanUA) [inline dockerfiles/alpine/dashboard/Dockerfile]: change the comment to suit the OS

File: dockerfiles/centos/worker/Dockerfile (hunk 1 above)
Comment (chirangaalwis) [reply dockerfiles/centos/worker/Dockerfile]: Done in commit https://github.com/wso2/docker-sp/pull/89/commits/c9dca5ea2e77496869f522594b89db2af76a1a9b.

File: dockerfiles/centos/manager/Dockerfile (hunk 2 above)
Comment (chirangaalwis) [reply dockerfiles/centos/manager/Dockerfile]: Done in commit https://github.com/wso2/docker-sp/pull/89/commits/c9dca5ea2e77496869f522594b89db2af76a1a9b.

File: dockerfiles/centos/editor/Dockerfile (hunk 3 above)
Comment (chirangaalwis) [reply dockerfiles/centos/editor/Dockerfile]: Done in commit https://github.com/wso2/docker-sp/pull/89/commits/c9dca5ea2e77496869f522594b89db2af76a1a9b.

File: dockerfiles/centos/dashboard/Dockerfile (hunk 4 above)
Comment (chirangaalwis) [reply dockerfiles/centos/dashboard/Dockerfile]: Done in commit https://github.com/wso2/docker-sp/pull/89/commits/c9dca5ea2e77496869f522594b89db2af76a1a9b.

File: dockerfiles/alpine/worker/Dockerfile (hunk 5 above)
Comment (chirangaalwis) [reply dockerfiles/alpine/worker/Dockerfile]: Done in commit https://github.com/wso2/docker-sp/pull/89/commits/c9dca5ea2e77496869f522594b89db2af76a1a9b.

File: dockerfiles/alpine/manager/Dockerfile (hunk 6 above)
Comment (chirangaalwis) [reply dockerfiles/alpine/manager/Dockerfile]: Done in commit https://github.com/wso2/docker-sp/pull/89/commits/c9dca5ea2e77496869f522594b89db2af76a1a9b.

File: dockerfiles/alpine/dashboard/Dockerfile (hunk 8 above)
Comment (chirangaalwis) [reply dockerfiles/alpine/dashboard/Dockerfile]: Done in commit https://github.com/wso2/docker-sp/pull/89/commits/c9dca5ea2e77496869f522594b89db2af76a1a9b.

**Pull request merged**